shared/teams/render.js.flow

```diff
@@ -0,0 +1,10 @@
+// @flow
+import * as React from 'react'
+
+export type Props = {
```

Comment:
we don't need the flow file if there's a unified js file for desktop and native

Reply:
Done.